src/garage/tf/algos/reps.py

```diff
@@ -256,34 +256,36 @@ def _build_inputs(self):
             ]
 
             with tf.name_scope('flat'):
-                obs_flat = flatten_batch(obs_var, name='obs_flat')
-                action_flat = flatten_batch(action_var, name='action_flat')
-                reward_flat = flatten_batch(reward_var, name='reward_flat')
-                valid_flat = flatten_batch(valid_var, name='valid_flat')
-                feat_diff_flat = flatten_batch(
+                obs_flat = flatten_batch_tensor(obs_var, name='obs_flat')
+                action_flat = flatten_batch_tensor(action_var,
+                                                   name='action_flat')
+                reward_flat = flatten_batch_tensor(reward_var,
+                                                   name='reward_flat')
+                valid_flat = flatten_batch_tensor(valid_var, name='valid_flat')
+                feat_diff_flat = flatten_batch_tensor(
                     feat_diff,
                     name='feat_diff_flat')  # yapf: disable
-                policy_state_info_vars_flat = flatten_batch_dict(
+                policy_state_info_vars_flat = flatten_batch_tensor_dict(
                     policy_state_info_vars,
                     name='policy_state_info_vars_flat')  # yapf: disable
-                policy_old_dist_info_vars_flat = flatten_batch_dict(
+                policy_old_dist_info_vars_flat = flatten_batch_tensor_dict(
                     policy_old_dist_info_vars,
                     name='policy_old_dist_info_vars_flat')
 
             with tf.name_scope('valid'):
-                reward_valid = filter_valids(
+                reward_valid = filter_valids_tensor(
                     reward_flat,
                     valid_flat,
                     name='reward_valid')   # yapf: disable
-                action_valid = filter_valids(
+                action_valid = filter_valids_tensor(
                     action_flat,
                     valid_flat,
                     name='action_valid')    # yapf: disable
-                policy_state_info_vars_valid = filter_valids_dict(
+                policy_state_info_vars_valid = filter_valids_tensor_dict(
                     policy_state_info_vars_flat,
                     valid_flat,
                     name='policy_state_info_vars_valid')
-                policy_old_dist_info_vars_valid = filter_valids_dict(
+                policy_old_dist_info_vars_valid = filter_valids_tensor_dict(
```

Comment:
is `policy_old_dist_info_vars` used? AKAIK this is only used to calculate KL divergence and/or log_likelihood under the old policy.

Reply:
This is being used - I only modifiy `npo` and this is `reps`. I only renamed the helper function here.